Menu/MenuBuilder.php

```diff
@@ -129,14 +78,4 @@ public function createSidebarMenu()
 
         return $event->getMenu();
     }
-
-    /**
-     * Sets the request the service.
-     *
-     * @param Request $request
-     */
-    public function setRequest(Request $request = null)
```

Comment:
Unless @afurculita does something for this, we need to change the milestone @core23 @Soullivaneuh @OskarStark